src/apis/rembrandt/lib/ResourceInstance.ts

```diff
@@ -0,0 +1,26 @@
+import { Serializer } from 'jsonapi-serializer';
+import Resource from '@/apis/jsonapi/Resource';
+import CRUDResource from '@/apis/jsonapi/CRUDResource';
+
+export interface ResourceInstanceAttribute {
+  name: string;
+  value: any;
+}
+
+export interface ResourceInstance extends Resource {
+  resourceType?: string;
+  attributes: ResourceInstanceAttribute[];
+}
+
+const serializer = new Serializer('resourceInstance', {
+  id: '_id',
```

Comment:
In https://github.com/bptlab/rembrandt-frontend/blob/abc80a00ee94b8522603d46b09de0ca85bba47d9/src/apis/jsonapi/Resource.ts#L2
you defined the ID to be named `id`, now you try to serialize the attribute named `_id`.
Does it work, even though the attribute was not defined with an underscore?

Reply:
Good point, the attribute is now defined with `_id`.